### PR TITLE
Milestone 4: Desktop version of my portfolio

### PR DESCRIPTION
**Pull request by Favourezeugwa**
I used media query to make my portfolio responsive and to  layout my desktop version 
Achieved this by using a min-width of 768px to style.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -82,7 +102,7 @@ <h1 class="project-heading">Projects</h1>
           class="project-image"
           alt="laptop"
         />
-        <div class="project-info">
+        <div class="project-info cont2">
           <p class="project-info-name">Project name goes here</p>
           <ul class="project-languages">
             <li>HTML/CSS</li>
```
Comment (zhadier) [inline index.html:108]: - [ ] Your second Project container seems to have introduced a bug and it's causing the info section to be out of place as you can see in the image below

> Yours
image

> Figma
image

Kindly fix this visual bug ✌️ . The rest of the design seems almost perfect 👍

File: index.html [hunk 2]
```diff
@@ -22,23 +22,43 @@
...
+      <nav>
+        <!-- Desktop nav-link starts-->
+        <ul id="header-nav">
+          <li><a href="index.html">Hello</a></li>
+          <li><a href="index.html">Portfolio</a></li>
+          <li><a href="index.html">About</a></li>
+          <li><a href="index.html">Contact</a></li>
+        </ul>
+        <!-- Desktop nav-link ends -->
+        <img src="./images/ic_menu.svg" alt="menu icon" id="menuimage" />
```
Comment (zhadier) [inline index.html]: - [ ] Your hamburger icon has disappeared from the mobile version as you can see below

image

Kindly make sure the mobile version stays the same even if you update the desktop version, such bugs are common occurrences during responsive development so please check it out an fix it.

**Review by zhadier — CHANGES_REQUESTED** [review]
# Hi @Favourezeugwa  👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
✔️ No linter errors
✔️ Github flow was set up correctly
✔️ Professional pull request title and description
✔️ You have done well matching your design to the original Figma design
✔️ You have used CSS grid and flexbox well for aligning the elements
✔️ Your design is looking absolutely 🔥 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question @zhadier so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OLIPLICHE — APPROVED** [review]
Hi @Favourezeugwa , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**